style.css

```diff
@@ -93,6 +107,16 @@ header {
 
 .design-images {
   position: relative;
+  display: initial;
+  width: 100vw;
+}
```

Comment:
- [ ] The about section in the desktop version works perfectly great job 👌💯, but I noticed in the mobile version that the background image doesn't work as expected, so kindly fix it 👌

![1](https://user-images.githubusercontent.com/45231276/177729177-a7b1f222-240e-4330-8745-e10f1cd38078.PNG)

- [ ] The same issue in the footer section 🙋‍♂️

![3](https://user-images.githubusercontent.com/45231276/177729365-76313f5d-3565-447a-9359-e49091843792.PNG)
 
> #### 💡 I think this problem happened because you added `display: initial` in this class `.design-images`, so I suggest removing this property from here 🤝

Reply:
Thanks for your feedback. It's really helpful

Reply:
Thank you